conf/application.conf

```diff
@@ -103,6 +103,13 @@ microservice {
       ducrPart: enabled
       betaBanner: enabled
     }
+
+    contact-frontend {
+      protocol = http
+      host = localhost
+      port = 9250
+      serviceIdentifier = "customs-movements-frontend"
```

Comment:
What do you think about naming this `serviceId`, as it is in contact-frontend? I am asking because I'd probably stay with the same name.